Scripts/Script0009_Add_Roles.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE ett_employee 
+ADD COLUMN Role VARCHAR(200) NOT NULL DEFAULT 'Developer'
```

Comment:
should we keep the names lowercase to be consistent? or is it not case sensitive?

Reply:
in the db it shows up as lowercase. that is still a typo on my part though I'll fix it